File: Submissions/#0429_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n, k;
  cin >> n >> k;

  string s;
  cin >> s;
  
  vector<int> color(26);

  for (int i = 0; i < n; i++) {
    color[s[i] - 'a']++;
  }

  string answer = "YES";

  for (int i = 0; i < 26; i++) {
    if (k < color[i]) {
      answer = "NO";
    }
  }

  cout << answer << endl;

}

File: Submissions/#0429_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();
    int k = sc.nextInt();
    String S = sc.next();
    sc.close();

    char[] s = S.toCharArray();
    int[] color = new int[26];

    for (int i = 0; i < n; i++) {
      color[s[i] - 'a']++;
    }
  
    String answer = "YES";
  
    for (int i = 0; i < 26; i++) {
      if (k < color[i]) {
        answer = "NO";
      }
    }
  
    System.out.println(answer);

  }

}